systems/data-plan/base-rate/pkg/validations/validations.go

```diff
@@ -0,0 +1,41 @@
+package validationErrors
```

Comment:
Also, sorry,  one last thing that I've failed to mention in our session: You need now to add a bunch of tests  for your repo functions, server rpcs and utils functions.

Reply:
Yea, We're planning to do that in separate PR.

Reply:
Okay, Got it!